paddlets/datasets/tsdataset.py

```diff
@@ -357,7 +357,7 @@ def get_index_at_point(
     def split(
         self, 
         split_point: Union[pd.Timestamp, str, float, int], 
-        after=True
+        after: Optional[bool]=True
```

Comment:
Line360 copy should Not be Optional, refers to https://docs.python.org/3/library/typing.html#typing.Optional